tools/eval.py

```diff
@@ -143,6 +144,9 @@ def main():
     if cfg.get('oss_io_config', None) is not None:
         io.access_oss(**cfg.oss_io_config)
 
+    # dynamic adapt mmdet models
```

Comment:
this file has related issue: #53   

we may remove this file, but it seems pose evaluation is supported in this file

Reply:
yes, pose is  supported,  but it‘s ok to just keep one